### PR TITLE
Update hyperframes to use state

### PR DESCRIPTION
**Pull request by briandoyle81CB**
**What changed? Why?**
Updated hyperframes tutorial to use the state property and teach it's use

**Notes to reviewers**

**How has it been tested?**

**Does this PR add a new token to the bridge?**

- [x] No, this PR does not add a new token to the bridge
- [ ] I've confirmed this token doesn't use a bridge override
- [ ] I've confirmed this token is an OptimismMintableERC20

Please include evidence of both confirmations above for your reviewers.

### CODE REVIEW

File: apps/base-docs/docs/building-with-base/guides/hyperframes.md [hunk 1]
```diff
@@ -2,10 +2,17 @@
 title: 'HyperFrames'
 description: Learn how to make cross-linked hyperframes in an organized manner.
 hide_table_of_contents: false
+keywords: [farcaster, frames, farcaster frames, hyperframes, hyper frames, state, frame state, Base]
 ---
 
 [Frames] on [Farcaster] are getting more complex. Developers are now building interactions that require a handful, or even dozens of frames in response to various user states, inputs, and actions. HyperFrames are a system to organize navigation for large numbers of frames, using [OnchainKit]. In this tutorial, we'll use making the navigation for an [old-school adventure game] fully in a frame. You can use this same technique for many other intents with your Frames, such as games, stores, customized mints, etc.
 
+:::info
```
Comment (taycaldwell) [inline apps/base-docs/docs/building-with-base/guides/hyperframes.md:10]: Idk if this call out is necessary. How far in the future would we remove it?

File: apps/base-docs/docs/building-with-base/guides/hyperframes.md [hunk 2]
```diff
@@ -305,13 +347,20 @@ frames['shack'] = {
     input: {
       text: 'What is the password?',
     },
-    postUrl: `${NEXT_PUBLIC_URL}/api/frame?frame=shack`,
+    state: { frame: 'shack' },
+    postUrl: `${NEXT_PUBLIC_URL}/api/frame?frame`,
```
Comment (taycaldwell) [inline apps/base-docs/docs/building-with-base/guides/hyperframes.md:351]: Should this just be:
```
postUrl: `${NEXT_PUBLIC_URL}/api/frame`,
```

?

File: apps/base-docs/docs/building-with-base/guides/hyperframes.md [hunk 3]
```diff
@@ -328,13 +377,16 @@ frames['shack-bad-password'] = {
     input: {
       text: 'Try again. What is the password?',
     },
-    postUrl: `${NEXT_PUBLIC_URL}/api/frame?frame=shack-bad-password`,
+    state: { frame: 'shack-bad-password' },
+    postUrl: `${NEXT_PUBLIC_URL}/api/frame?frame`,
```
Comment (taycaldwell) [inline apps/base-docs/docs/building-with-base/guides/hyperframes.md:381]: Should this just be:

```
postUrl: `${NEXT_PUBLIC_URL}/api/frame`,
```

?

File: apps/base-docs/docs/building-with-base/guides/hyperframes.md (hunk 1 above)
Comment (briandoyle81CB) [reply apps/base-docs/docs/building-with-base/guides/hyperframes.md:10]: I'll delete it.  It's redundant with the warning that these are in flux now anyway.

**Review by taycaldwell — APPROVED** [review]

**Pull request merged**